backend/src/helpers/secrets.ts

```diff
@@ -323,38 +297,39 @@ export const createSecretHelper = async ({
   secretCommentCiphertext,
   secretCommentIV,
   secretCommentTag,
-  secretPath = "/",
+  secretPath = "/"
 }: CreateSecretParams) => {
-
   const secretBlindIndex = await generateSecretBlindIndexHelper({
     secretName,
-    workspaceId: new Types.ObjectId(workspaceId),
+    workspaceId: new Types.ObjectId(workspaceId)
   });
 
   // if using service token filter towards the folderId by secretpath
   if (authData.authPayload instanceof ServiceTokenData) {
-    const { secretPath: serviceTkScopedSecretPath } = authData.authPayload;
-    if (secretPath !== serviceTkScopedSecretPath) {
+    const { scopes: tkScopes } = authData.authPayload;
+    const validScope = tkScopes.find(
+      (scope) =>
+        picomatch.isMatch(secretPath, scope.secretPath, { strictSlashes: false }) &&
```

Comment:
i think this should `secretPath || "/"` ? Because for the other ones i see you have made it so that if no path is defined then the secret is created on root

Reply:
Oh i see, secretPath is already defaut to '/'